Firmware/Marlin.h

```diff
@@ -232,6 +232,7 @@ extern int feedmultiply;
 extern int extrudemultiply; // Sets extrude multiply factor (in percent) for all extruders
 extern float extruder_multiplier[EXTRUDERS]; // reciprocal of cross-sectional area of filament (in square millimeters), stored this way to reduce computational burden in planner
 extern float current_position[NUM_AXIS] ;
+extern float pause_position[3];
```

Comment:
Please remove this. `pause_position` is not used any other files than `Marlin_main.cpp`